testdata/OpenSSL/build.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+set -e
+set -u
+
+password=123456
+
+openssl req -x509 \
+            -sha256 -days 3560 \
+            -newkey rsa:2048 \
+            -nodes \
+            -subj "/CN=test.hazelcast.com/C=US/L=San Fransisco" \
+            -keyout rootCA.key -out rootCA.crt
+
+cat > csr.conf <<EOF
+[ req ]
+default_bits = 2048
+prompt = no
+default_md = sha256
+req_extensions = req_ext
+distinguished_name = dn
+
+[ dn ]
+C = US
+ST = California
+L = San Fransisco
+O = Hazelcast Test
+OU = Hazelcast Go Client Test
+CN = test.hazelcast.com
+
+[ req_ext ]
+subjectAltName = @alt_names
+
+[ alt_names ]
+DNS.1 = test.hazelcast.com
+DNS.2 = www.test.hazelcast.com
+IP.1 = 192.168.1.5
+IP.2 = 192.168.1.6
```

Comment:
How about saving this to `csr.conf` instead and load it?

Reply:
The command already exports the configuration as file to the current directory, I think there is no need for applying extra file operation, if you are okay with that.

Reply:
What I meant was moving that content to the separate `csr.conf` file and load it from the script. Is there an advantage of keeping it in the script?